src/snowflake/snowpark/modin/pandas/base.py

```diff
@@ -1986,15 +1986,14 @@ def idxmin(self, axis=0, skipna=True, numeric_only=False):  # noqa: PR01, RT01,
             )
         )
 
+    @base_not_implemented()
```

Comment:
what is this annotation here ?

Reply:
This was added in https://github.com/snowflakedb/snowpark-python/pull/1472 to automatically raise NotImplementedError with the appropriate API name.

Reply:
I see.